src/components/clusterConfiguration/ReviewValidations.tsx

```diff
@@ -67,7 +67,10 @@ const FailingValidation: React.FC<{
 
   let fix;
   const step = findValidationFixStep({ id: validation.id, clusterGroup, hostGroup });
-  if (step) {
+  if (step === 'review') {
+    // no sooner step, so the user can not do anything about it ...
+    fix = 'Please wait till all validations are finished.';
```

Comment:
This is not exactly true as `all-hosts-are-ready-to-install` validation is failed and won't get updated.

Reply:
Asking backend to confirm, but so far that was my understanding, so the fix would be ok. Let's see their response.

Reply:
confirmed, just temporary flow till the validation is re-executed